src/fosslight_dependency/package_manager/Gradle.py

```
@@ -39,77 +39,134 @@ def parse_oss_information(self, f_name):
+
+            if json_format == "list":
+                module_licenses_raw = d.get("moduleLicenses", [])
+                license_names = []
+                for lic in module_licenses_raw:
+                    lic_name = lic.get("name") or ""
+                    lic_url = lic.get("url") or ""
+
+                    final_license_name = normalize_license_name_from_name(lic_name) if lic_name else None
+
+                    multi_license_keywords = ["dual", "multi", "and", "or", "+"]
+                    if not lic_name.lower() in multi_license_keywords and lic_url:
```

Ruff (0.15.15)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)